tools/sim/tests/test_metadrive_integration.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
```

Comment:
if we're going to abstract all this stuff for CARLA, I don't want to duplicate the test like this. however, it would also be fine to add the metadrive bridge + tests without touching CARLA at all. the bar will be higher for abstracting the CARLA stuff

Reply:
Yes, we can combine the test.

Reply:
Merged the test. Should we add it to CI now?

Would prefer to incrementally enable in subsequent PR (next one perhaps).